cirq-core/cirq/circuits/circuit.py

```diff
@@ -1323,11 +1343,11 @@ def save_qasm(
         self._to_qasm_output(header, precision, qubit_order).save(file_path)
 
     def _json_dict_(self):
-        return protocols.obj_to_dict_helper(self, ['moments', 'device'])
+        return protocols.obj_to_dict_helper(self, ['moments'])
```

Comment:
This confuses me a bit: we still support `device` everywhere, except for JSON? It seems like this and `_from_json_dict_` should continue supporting `device` until it's removed everywhere else as well.

Reply:
Good point, got a little ahead of myself there. Went ahead and put device back in json. We can cut it and add the {repr,json}_inward once things are officially cut when the deprecation cycle is over.